config.json

```diff
@@ -63,8 +63,8 @@
       "unlocked_by": "resistor-color",
       "difficulty": 1,
       "topics": [
-        "enums",
-        "arrays"
+        "arrays",
+        "enums"
```

Comment:
```suggestion
        "enums",
        "arrays"
```

Reply:
same for all the rest of the changes in this file (apart from as related to this exercise). Not able to propose changes for all of them so it will require a local side change.